01-Fundamentals-part-2/assignments2.js

```diff
@@ -21,4 +21,24 @@ console.log("\n");
 console.log("Testing Data 2:");
 avgDolphins = calcAverage(85, 54, 71);
 avgKoalas = calcAverage(23, 34, 27);
-checkWinner(avgDolphins, avgKoalas);
+checkWinner(avgDolphins, avgKoalas);*/
+
+function calcTip(billValue) {
+  if (billValue > 50 || billValue < 300) {
```

Comment:
hmm, if understand the problem correctly, if the bill is between 50 and 300, the tip is 15%, else is 20%, or? Then you used the wrong logical operator. Check: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_OR vs https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_AND